src/trans/internal/trgtol_mod.F90

```diff
@@ -149,7 +149,7 @@ SUBROUTINE TRGTOL_PROLOG(KF_FS,KF_GP,KVSET,&
 
 
 USE PARKIND1  ,ONLY : JPIM     ,JPRB
-USE YOMHOOK   ,ONLY : LHOOK,   DR_HOOK, JPHOOK
```

Comment:
I don't think DR_HOOK and JPHOOK are used in this routine at all.

Reply:
Done